src/app/projects/state/projects.reducer.ts

```diff
@@ -7,30 +7,62 @@ const DEFAULT_PROJECT: ProjectEntities = {
   default: createNewProject('default')
 };
 
-function projectsStateReducer(state: ProjectsState, action: ProjectsActions) {
+export const favoritesReducer = (favoriteEntities: FavoriteEntities, action: ProjectsActions) => {
```

Comment:
Let's convert this into a proper function.

Reply:
Why did the order change of the functions? It's a little hard to review to see if something changed.